src/quo/components/wallet/swap_input/view.cljs

```diff
@@ -168,10 +169,10 @@
              :i/reorder])])]
       [divider-line/view]
       [rn/view
-       {:style (style/row-2 (or (not pay?) loading?))}
+       {:style (style/row-2 (or (not max-button-enabled?) loading?))}
        (when-not loading?
          [:<>
-          (when pay?
+          (when max-button-enabled?
```

Comment:
Will this affect other tokens? For all tokens except ETH, we can keep the max functionality as it is.

Reply:
@shivekkhurana it won't effect other token as i recorded in the video. it just check if there the press func passed to component to display the max button. unless it won't show it.